### DISCUSSION TITLE
Update from 3.0 to 4.5

### DISCUSSION THREAD

**Discussion by meszat2** [OpenIdentityPlatform/OpenDJ#240] [Q&A] [answered]
Hi!

How can I upgrade from OpenDJ 3.0 to OpenDJ 4.5?

Have I to replace binary only, or have I to unload and reload my records.

Regards,
Attila

**Comment by polybas** [comment, marked as answer]
- In OpenDJ 3.0 create Full Backup.
- Stop OpenDJ 3.0
- Install OpenDJ 4.5 in new directory
- Start OpenDJ 4.5
- Restore from Backup (select path of Full Backup)

**Reply by meszat2** [reply]
Thanks

**Comment by vharseko** [comment]
1. backup  
2. replace /bin /lib
3. run ./upgrade
